Confus/Confus/Game.cpp

```diff
@@ -61,6 +66,13 @@ namespace Confus
                 selector = m_Device->getSceneManager()->createTriangleSelectorFromBoundingBox(node);
                 break;
             case irr::scene::ESNT_MESH:
+                if(node->getID() == 2) {
+                    selector = sceneManager->createTriangleSelector(((irr::scene::IMeshSceneNode*)node)->getMesh(), node);
```

Comment:
This if doesn't seem to do anything (the conents of the if are the same as the else) (also, changing the formatting would be apreciated)